src/api/quantum/block/Block.java

```diff
@@ -12,6 +15,8 @@
  * @author link
  */
 @WIP
+@SideOnly(Side.PROXY)
+@Universal
```

Comment:
For consistency (and readability) sake, might do better to have an order using annotations, probably shortest to longest (just suggesting)